components/uniset/src/uniset_const.rs

```diff
@@ -0,0 +1,1928 @@
+#![allow(clippy::unreadable_literal, dead_code)]
+
+use crate::uniset::UnicodeSet;
+
+/// Upper
+pub fn upper() -> UnicodeSet {
+    UnicodeSet::from_inversion_list(vec![
```

Comment:
Nit: can you make all of these hex?
Ideally zero-padded to 4 digits (6 beyond BMP)

Reply:
(for all of these kind of initializations, of course)

Reply:
The file has been moved from the `uniset` component to the `provider_ppucd` component. More importantly, each property getter fn now uses a data provider argument, loads the inversion list from it using the appropriate resource key (data key) for the property represented by the gettern fn, and converts that inversion list into a UnicodeSet.